### ISSUE TITLE
Repeated Ctrl+B should undo formatting

### ISSUE THREAD

**Issue by borekb** [neilsustc/vscode-markdown#226]
Ctrl+B should ideally be context-aware: if I'm in plain text, insert `**` markup; if I'm in a bold text, remove it. The current behavior just keeps adding `**`.

Similarly for Ctrl+I.

(I'd also like this for links as removing link markup is especially annoying; but Ctrl+L is currently implemented as a snippet only so that's probably not possible at the moment.)

**Comment by neilsustc** [comment]
I don't understand. We already had it, do we?

![style](https://user-images.githubusercontent.com/7588612/42230064-a35e6844-7f1a-11e8-8453-ed744d7d4d78.gif)

(the same for selected text)

**Comment by borekb** [comment]
Oh, it's because I tried on a bold text that spans more words. I didn't realize this is already working for single-word formatting. Sorry!

**Comment by neilsustc** [comment]
Then you need to select the text manually 😂

**Comment by borekb** [comment]
Yeah, no big deal, Caret is more intelligent about this but I understand that this is a relatively unimportant feature, as deleting `**` or `_` is not that difficult.

I'd seriously be interested in "reverse Ctrl+L" as that annoys me quite a bit :)

**Comment by neilsustc** [comment]
I am a little bit curious about why you need to reverse ctrl+l so frequently 😂

And I will be busy for the whole month so there might not be enough time for things other than minor fixes. I am going to mark this as "help wanted"
